flink-table-store-hive/flink-table-store-hive-catalog/src/main/java/org/apache/flink/table/store/hive/HiveCatalog.java

```diff
@@ -408,12 +411,16 @@ private Lock lock(ObjectPath tablePath) {
         return Lock.fromCatalog(lock, tablePath);
     }
 
-    static IMetaStoreClient createClient(HiveConf hiveConf) {
+    static IMetaStoreClient createClient(HiveConf hiveConf, String clientClassName) {
         IMetaStoreClient client;
         try {
             client =
                     RetryingMetaStoreClient.getProxy(
-                            hiveConf, tbl -> null, HiveMetaStoreClient.class.getName());
+                            hiveConf,
```

Comment:
We can verify multiple hive versions. I don't know if it is good to all supported versions.

Reply:
I've tested Hive 2.1, 2.2, 2.3 and 3.1. They are all supported.